.swiftlint.yml

```diff
@@ -22,7 +22,6 @@ opt_in_rules:
   - self_binding
   - self_in_property_initialization
   - shorthand_optional_binding
-  - sorted_imports
```

Comment:
Is imports sorting done by SwiftFormat now?

Reply:
Yes they kind of interfere with each other otherwise... `SwiftFormat` will place all `testable` after regular imports but it doesn't take care of adding a space between them. If there is no space between them `SwiftLint` will try to sort them alphabetically so `testable` and regular get mixed up. I couldn't find an easy way to fix this

Reply:
maybe disable the swiftlint rule @samwyndham ?

Reply:
@netbe What do you mean. You mean something else than deleting the rule?

Reply:
By removing this line from opt_in_rules it's already disabled.